translations/es-es.json

```diff
@@ -0,0 +1,30 @@
+{
+  "colors_group_label": "Colores",
+  "brand_color_description": "El color propio de la marca para los elementos importantes de navegación",
+  "brand_color_label": "Color de marca",
+  "brand_text_color_description": "El color de la marca cuando está activa y cuando se le pasa el cursor",
+  "brand_text_color_label": "Color de texto de la marca",
+  "text_color_description": "El color del texto para los elementos de cuerpo y encabezado",
+  "text_color_label": "Color de texto",
+  "link_color_description": "El color del texto para los elementos de vínculos",
+  "link_color_label": "Color de vínculos",
+  "background_color_description": "El color de fondo del Centro de ayuda",
+  "background_color_label": "Color de fondo",
+  "fonts_group_label": "Fuentes",
+  "heading_font_description": "La fuente para los encabezados",
+  "heading_font_label": "Fuente de encabezado",
+  "text_font_description": "La fuente para el texto del cuerpo",
+  "text_font_label": "Fuente de texto",
+  "brand_group_label": "Marca",
+  "logo_description": "El logotipo de la compañía",
+  "logo_label": "Logotipo",
+  "favicon_description": "Icono mostrado en la barra de dirección del navegador",
+  "favicon_label": "Favicon",
+  "images_group_label": "Imágenes",
+  "homepage_background_image_description": "La imagen hero de la página principal",
```

Comment:
@zendesk/i18n "Imagen hero" doesn't really mean anything, maybe something like "Imagen principal de la página de inicio" would be more understandable.

Reply:
Thanks, we did not even know what it meant in English. We will add to glossary and update all strings (and I bet all languages)

Reply:
@olenavelykanova